persistence-modules/hibernate-mapping/src/main/java/com/baeldung/hibernate/wherejointable/Group.java

```diff
@@ -0,0 +1,85 @@
+package com.baeldung.hibernate.wherejointable;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
+import javax.persistence.ManyToMany;
+
+@Entity(name = "e_group")
+public class Group {
+
+    @Id
+    @GeneratedValue
+    private Long id;
+
+    private String name;
+
+    @ManyToMany(mappedBy = "groups")
+    private List<User> users = new ArrayList<>();
+
+    public Group() {
+
```

Comment:
Let's remove the redundant blank line, or put in a comment about why there's a default constructor as well as a parameterized one.

Reply:
You are right, we do not need to define default constructor here so I will remove them for both Group and User classes.